share/man/man4/gpioled.4

```diff
@@ -70,7 +70,20 @@ Which pin on the GPIO interface to map to this instance.
 Please note that this mask should only ever have one bit set
 (any other bits - i.e., pins - will be ignored).
 .It Va hint.gpioled.%d.invert
-If set to 1, the pin will be set to 0 to light the LED, and 1 to clear it.
+Use pin inversion. If set to 1, the pin will be set to 0 to light the LED, and 1
+to clear it.
+.It Va hint.gpioled.%d.invmode
+When pin inversion is requested, whether or not to use hardware support. Must be
```

Comment:
I think it would flow better like this:
"Whether or not to use hardware support when pin inversion is requested"